CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 ### Removed
 - Python 3.5 and 3.6 support
+- Django 2.2, 3.0, and 3.1 support
```

Comment:
fixed.  dealing with CHANGELOG conflicts isn't very contributor friendly. It would be nice if these were a part of the release process, either manually aggregated by the maintainers or by something like [semantic-release](https://python-semantic-release.readthedocs.io/en/latest/).